src/test/resources/mockdata/user_collection.json

```diff
@@ -31,7 +31,9 @@
         "firstName": "Mockey",
         "email": "mock@biglibrary.org",
         "phone": "2125551212"
-      }
+      },
+      "externalSystemId": "123",
+      "barcode": "test123"
```

Comment:
Do we have some test to check that these fields are populated?

Reply:
we dont have any direct test for checking these fields , because method is private and directly been used inside kafkaService.